edx_proctoring/backends/software_secure.py

```diff
@@ -80,7 +80,7 @@ def register_exam_attempt(self, exam, context):
 
         if status not in [200, 201]:
             err_msg = (
-                u'Could not register attempt_code = {attempt_code}. '
+                u'Could not register attempt_code={attempt_code}. '
```

Comment:
One nit question: why `u` prefix was left? It seems that it could be removed here as well.